_posts/2015-03-23-design-patterns-part-1.markdown

```diff
@@ -0,0 +1,131 @@
+---
+layout:post
+title:'Design Patterns, Fast Tests, and Robust Software, Part I'
```

Comment:
Apparently single quotes aren't kosher for jekyll? The errors go away once I use double quotes for the title.